### PR TITLE
feat: udf onnx config placeholder

### PR DESCRIPTION
**Pull request by chloeminkyung**
```yaml
udfs:
   name: is_fraudulent
   config: !Onnx
     path: ./models/model_file 
```

### CODE REVIEW

File: dozer-types/src/models/config.rs [hunk 1]
```diff
@@ -75,6 +76,11 @@ pub struct Config {
     /// Dozer Cloud specific configuration
     #[serde(skip_serializing_if = "Option::is_none")]
     pub cloud: Option<Cloud>,
+
+    #[prost(message, optional, tag = "15")]
+    /// UDF specific configuration (eg. !Onnx)
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub udf: Option<UdfConfig>,
```
Comment (chubei) [inline dozer-types/src/models/config.rs:83]: `udfs: Vec<UdfConfig>`

File: dozer-types/src/models/udf_config.rs [hunk 2]
```diff
@@ -0,0 +1,30 @@
+use crate::serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Eq, PartialEq, Clone, prost::Message)]
+pub struct UdfConfig {
+    #[prost(string, tag = "1")]
+    /// name of the model function
+    pub name: String,
+    #[prost(oneof = "UdfType", tags = "2")]
+    #[serde(default = "default_udf_type")]
```
Comment (chubei) [inline dozer-types/src/models/udf_config.rs:9]: We shouldn't `default` this field

File: dozer-types/src/models/udf_config.rs [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+#[derive(Serialize, Deserialize, Eq, PartialEq, Clone, ::prost::Message)]
+pub struct OnnxConfig {
+    #[prost(string)]
+    /// path to the model file
+    pub path: String,
+}
+
+fn default_udf_type() -> Option<UdfType> {
```
Comment (chubei) [inline dozer-types/src/models/udf_config.rs:28]: `OnnxConfig::default` doesn't make sense

**Review by chubei — CHANGES_REQUESTED** [review]

File: dozer-types/src/models/config.rs (hunk 1 above)
Comment (chloeminkyung) [reply dozer-types/src/models/config.rs]: addressed

File: dozer-types/src/models/udf_config.rs (hunk 2 above)
Comment (chloeminkyung) [reply dozer-types/src/models/udf_config.rs]: addressed

File: dozer-types/src/models/udf_config.rs (hunk 3 above)
Comment (chloeminkyung) [reply dozer-types/src/models/udf_config.rs]: addressed

Comment (chloeminkyung) [reply dozer-types/src/models/udf_config.rs]: addressed

**Review by chubei — APPROVED** [review]

**Pull request merged**